nixos/modules/services/misc/nitter.nix

```diff
@@ -155,6 +160,22 @@ in
           description = lib.mdDoc "Use base64 encoding for proxied media URLs.";
         };
 
+        enableRSS = mkEnableOption (lib.mdDoc "Set this to false to disable RSS feeds.");
+
+        enableDebug = mkEnableOption (lib.mdDoc "Enable request logs and debug endpoints.");
```

Comment:
`mkEnableOption` makes so the message is prefixed with `Whether to enable [...]` and adds a period.

So this would read `Whether to enable Enable requestlogs and debug endpoints..` (notice the two periods) for example.

Reply:
Thanks a lot! I was not aware of these details. It should now be fixed.